Topics/jupyter-notebook.md

```diff
@@ -0,0 +1,154 @@
+# Jupyter Notebook
+
+# What you will learn 
+In this guide you'll learn how to use the Jupyter Notebook and the integrated JupyterLab IDE. Jupyter is a free and open-source web application (in the form of a Python module) that allows you to create and share documents that contain code, equations, text and images incorporated.
+
+It's widely used in the Data Science process, in order to explore data, clean them, and experiment problem solutions.
+
+## Prerequisites
+- [Python Fundamentals](python-fundamentals.md)
+
+## Time to complete
+5 minutes.
+
+
+# Index
+ - [Why Jupyter Notebooks](#Why-Jupyter-Notebooks)
+ - [Learn to use it](#Learn-to-use-it)
+ - [Considerations](#Considerations)
+ - [Tips and Tricks](#Tips-and-Tricks)
+ - [Notebooks Examples](#Notebooks-Examples)
+ - [Conclusions](#Conclusions)
+
+
+Let's dive right in!
+ 
+### Why Jupyter Notebooks
+
+[Jupyter notebooks](https://jupyter.org/) are a way of embedding ready-to-execute python code with markdown documentation, images, and other such things, all in one file, as well as a program for presenting and running them. 
+
+They're nice if you want to demonstrate a short script to another person, particularly in a data science context, because of some built-in plotting niceties and the fact that you can run cells of code individually. 
+
+They are [widely used in Data Science](https://www.nature.com/articles/d41586-018-07196-1), because as we know the Data Science Process is an experimental and incremental process.
+
+There are several reasons to choose Jupyter Notebook for the first phases of the Data Science process (e.g. data exploration, data cleaning):
+
+- it's easy to write more than one line of the code before running. You do not need an editor on the side, crafting functions and then copying to the shell
+
+- results presentation is richer than console. It supports graphs, images, tables. Effectively anything that can be represented by html, css and js
+
+- if you make a mistake, you can trivially rerun that "cell" (chunk of code, nothing else) once you fix the mistake
+
+- you can install it on some more powerful machine and interact with it in your browser, so works for you if you're not super friendly with terminal stuff
+
+- you can export run results as html page. a run of a notebook is effectively a static thing. notebook consists of code + results of running the code.
```

Comment:
Okay this is similar to what I said earlier, try to see if you can combine the two together to give a more complete picture of the feature or benefit